apps/docs/scripts/test-template.ts

```diff
@@ -11,7 +11,7 @@ afterAll(() => {
 });
 
 // %TEST_ENVIRONMENT%
-test('%NAME%', async () => {
+test('%NAME%', { timeout: 15_000 }, async () => {
```

Comment:
The `wrong-resubmission` snippet test was consistently timing out. We might want to have a way of configuring this on a per-snippet basis.